modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -38,46 +38,48 @@ const getSelectedCells = (table: SugarElement<HTMLTableElement>, cells: SugarEle
   }));
 };
 
-const updateSimpleProps = (modifier: DomModifier, colModifier: DomModifier, data: CellData): void => {
-  modifier.setAttrib('scope', data.scope);
-  modifier.setAttrib('class', data.class);
-  modifier.setStyle('height', Utils.addPxSuffix(data.height));
-  colModifier.setStyle('width', Utils.addPxSuffix(data.width));
+const updateSimpleProps = (modifier: DomModifier, colModifier: DomModifier, data: CellData, shouldUse: (key: string) => boolean): void => {
+  if (shouldUse('scope')) {
```

Comment:
`shouldUse` is a little confusing here... maybe call this `shouldUpdate` instead, since that's what it's doing?